cc33xx-target-scripts/suite/trixie/debian/copyright

```diff
@@ -0,0 +1,90 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: cc33xx-target-scripts
+Upstream-Contact: Suhaas Joshi <s-joshi@ti.com>
+Source: https://git.ti.com/cgit/cc33xx-wlan/cc33xx-target-scripts/
+
+Files:     ap/ap_start.sh
+           ap/ap_stop.sh
+           conf/udhcpd.conf
+           conf/wpa_supplicant.conf
+           conf/wpa_supplicant2.conf
+           drv/am335x_sdio_rebind.sh
+           drv/am6x_sdio_rebind.sh
+           drv/antenna_selection.sh
+           drv/cc33xx_twt.sh
+           drv/diversity_enable.sh
+           drv/diversity_select_default_antenna.sh
+           drv/diversity_set_rssi_threshold.sh
+           drv/load_cc33xx.sh
+           drv/set_lsi.sh
+           drv/set_power_mode.sh
+           drv/unload_cc33xx.sh
+           sta/sta_add.sh
+           sta/sta_connect-ex-dhcp.sh
+           sta/sta_connect-ex.sh
+           sta/sta_connect.sh
+           sta/sta_remove.sh
+           sta/sta_start.sh
+           sta/sta_stop.sh
+Copyright: __NO_COPYRIGHT_NOR_LICENSE__
```

Comment:
All these should be covered under the license in the COPYING file, which looks like a standard 3-clause BSD.

Reply:
Its TI-TFL. So have pushed the (hopefully) corrected copyright.